index.html

```diff
@@ -0,0 +1,152 @@
+              fill="#505F79"
+            />
+          </svg>
+        </li>
+        <li class="social">
+          <svg
+            width="21"
+            height="20"
+            viewBox="0 0 21 20"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M14.061 3.03418C10.5722 3.03418 7.70523 5.86177 7.70523 9.37901V19.31C7.70523 19.6894 8.01611 19.9997 8.39608 19.9997H11.9885C12.3684 19.9997 12.6793 19.6894 12.6793 19.31V9.65487C12.6793 8.89625 13.3011 8.27556 14.061 8.27556C14.8209 8.27556 15.4427 8.89625 15.4427 9.65487V19.31C15.4427 19.6894 15.7536 19.9997 16.1336 19.9997H19.726C20.1059 19.9997 20.4168 19.6894 20.4168 19.31V9.37901C20.4168 5.89625 17.5843 3.03418 14.061 3.03418Z"
+              fill="#505F79"
+            />
+            <path
+              d="M2.90371 0C1.52202 0 0.416664 1.10345 0.416664 2.48276C0.416664 3.86207 1.52202 4.96552 2.90371 4.96552C4.2854 4.96552 5.39076 3.86207 5.39076 2.48276C5.39076 1.10345 4.2854 0 2.90371 0Z"
+              fill="#505F79"
+            />
+            <path
+              d="M4.69991 6.27539H1.10751C0.727545 6.27539 0.416664 6.58574 0.416664 6.96505V19.3099C0.416664 19.6892 0.727545 19.9995 1.10751 19.9995H4.69991C5.07988 19.9995 5.39076 19.6892 5.39076 19.3099V6.96505C5.39076 6.58574 5.07988 6.27539 4.69991 6.27539Z"
+              fill="#505F79"
+            />
+          </svg>
+        </li>
+        <li class="social">
+          <svg
+            width="21"
+            height="16"
+            viewBox="0 0 21 16"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M18.8208 1.88042L20.4167 0.337684V0H14.8883L10.9483 9.90989L6.46582 0H0.669156V0.337684L2.53332 2.60547C2.71499 2.77305 2.80999 3.01811 2.78582 3.26484V12.1768C2.84332 12.4977 2.73999 12.8278 2.51666 13.0611L0.416656 15.6328V15.9663H6.37082V15.6286L4.27082 13.0611C4.04332 12.8269 3.93582 12.5027 3.98166 12.1768V4.46821L9.20832 15.9705H9.81582L14.31 4.46821V13.6312C14.31 13.8728 14.31 13.9225 14.1533 14.0808L12.5367 15.6615V16H20.38V15.6623L18.8217 14.1204C18.685 14.016 18.6142 13.8408 18.6433 13.6707V2.33011C18.6142 2.15916 18.6842 1.984 18.8208 1.88042Z"
+              fill="#505F79"
+            />
+          </svg>
+        </li>
+        <li class="social">
+          <svg
+            width="19"
+            height="20"
+            viewBox="0 0 19 20"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
```

Comment:
- [ ] Kindly anusre that each icon item has a link (`a` tag) apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href` atribute.
```HTML
  <ul>
      <li>
        <a href="#">...</a>
      </li>
      <li>
        <a href="#">...</a>
      </li>
      ...
    </ul>
```